application/src/main/java/org/opentripplanner/routing/linking/VertexLinker.java

```diff
@@ -226,20 +227,38 @@ private DisposableEdgeCollection link(
     return tempEdges;
   }
 
+  public Set<StreetVertex> linkToSpecificStreetEdgesPermanently(
```

Comment:
I know, this class is not well documented, but let's not make it worse: can you describe why you want to select a specific edge?

Reply:
Because the centroid of that linear platform is on that edge

Reply:
Sorry, I meant to add Javadoc.